reference/sql/statements/split-region.md

```diff
@@ -168,6 +181,149 @@ region3  [("b", "2019-04-17 14:26:19") , ("c", "")                   )
+
+```sql
+split partition table t between (0) and (10000) regions 4;
+```
+
+再用 `SHOW TABLE REGIONS` 语法查看该表的 REGION 如下，会发现该表现在一共有 10 个 region，每个 partition 分别有 5 个 region，其中 4 个 region 是表的行数据，1 个 region 是表的索引数据。
+
+{{< copyable "sql" >}}
+
+```sql
+show table t regions;
+```
+
+```sql
++-----------+---------------+---------------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
+| REGION_ID | START_KEY     | END_KEY       | LEADER_ID | LEADER_STORE_ID | PEERS            | SCATTERING | WRITTEN_BYTES | READ_BYTES | APPROXIMATE_SIZE(MB) | APPROXIMATE_KEYS |
++-----------+---------------+---------------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
+| 1998      | t_1400_r      | t_1400_r_2500 | 2001      | 5               | 2000, 2001, 2015 | 0          | 132           | 0          | 1                    | 0                |
+| 2006      | t_1400_r_2500 | t_1400_r_5000 | 2016      | 1               | 2007, 2016, 2017 | 0          | 35            | 0          | 1                    | 0                |
+| 2010      | t_1400_r_5000 | t_1400_r_7500 | 2012      | 2               | 2011, 2012, 2013 | 0          | 35            | 0          | 1                    | 0                |
+| 1978      | t_1400_r_7500 | t_1401_       | 1979      | 4               | 1979, 1980, 1981 | 0          | 621           | 0          | 1                    | 0                |
+| 1982      | t_1400_       | t_1400_r      | 2014      | 3               | 1983, 1984, 2014 | 0          | 35            | 0          | 1                    | 0                |
+| 1990      | t_1401_r      | t_1401_r_2500 | 1992      | 2               | 1991, 1992, 2020 | 0          | 120           | 0          | 1                    | 0                |
+| 1994      | t_1401_r_2500 | t_1401_r_5000 | 1997      | 5               | 1996, 1997, 2021 | 0          | 129           | 0          | 1                    | 0                |
+| 2002      | t_1401_r_5000 | t_1401_r_7500 | 2003      | 4               | 2003, 2023, 2022 | 0          | 141           | 0          | 1                    | 0                |
+| 6         | t_1401_r_7500 |               | 17        | 4               | 17, 18, 21       | 0          | 601           | 0          | 1                    | 0                |
+| 1986      | t_1401_       | t_1401_r      | 1989      | 5               | 1989, 2018, 2019 | 0          | 123           | 0          | 1                    | 0                |
++-----------+---------------+---------------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
+```
+
+也可以给每个分区的索引切分 region ，如将索引 `idx` 的 [1000,10000] 范围切分成 2 个 region：
+
+{{< copyable "sql" >}}
+
+```sql
+split partition table t index idx between (1000) and (10000) regions 2;
```

Comment:
Split region for index functions on all the partitions rather than on a specified partition?

Reply:
yes,